content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,236 @@
+    $ kubectl get pod -n test-injection
+    {{< /text >}}
+
+    The output from the `get pod` command should show the following output. The `2/2` value means that
+    the webhook injected a sidecar into the example pod:
+
+    {{< text plain >}}
+    NAME    READY   STATUS    RESTARTS   AGE
+    nginx-app   2/2     Running   0          10s
+    {{< /text >}}
+
+1.  Check that the validation webhook is working:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-validation
+    $ kubectl apply -n test-validation -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: invalid-gateway
+    spec:
+      selector:
+        # DO NOT CHANGE THESE LABELS
+        # The ingressgateway is defined in install/kubernetes/helm/istio/values.yaml
+        # with these labels
+        istio: ingressgateway
+    EOF
+    {{< /text >}}
+
+    The output from the gateway creation command should show the following output. The error
+    in the output indicates that the validation webhook checked the gateway's configuration YAML file:
+
+    {{< text plain >}}
+    Error from server: error when creating "invalid-gateway.yaml": admission webhook "pilot.validation.istio.io" denied the request: configuration is invalid: gateway must have at least one server
+    {{< /text >}}
+
+## Show webhook configurations
+
+1.  Show the configurations of Galley and the sidecar injector with their default configuration names:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status
+    {{< /text >}}
+
+1.  Show the configuration of the sidecar injector if the webhook's configuration name is `istio-sidecar-injector`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status --validation=false --injection-config=istio-sidecar-injector
+    {{< /text >}}
+
+1.  Show the configuration of Galley if the webhook's configuration name is `istio-galley`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status --injection=false --validation-config=istio-galley
+    {{< /text >}}
+
+## Disable webhook configurations
+
+1.  Disable Galley and the sidecar injector when using their default webhook configuration names:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable
+    {{< /text >}}
+
+1.  Disable the sidecar injector if the webhook's configuration name is `istio-sidecar-injector`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable --validation=false --injection-config=istio-sidecar-injector
+    {{< /text >}}
+
+1.  Disable the configuration of Galley if the webhook's configuration name is `istio-galley`:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable --injection=false --validation-config=istio-galley
+    {{< /text >}}
+
+## Cleanup
+
+After completing this tutorial, you may delete the testing cluster created
```

Comment:
Please do not make a test cluster a requirement for tasks. Instead, make sure features can be enabled and disabled on existing deployments.

Reply:
Creating a test cluster is to safely try this feature without affecting existing deployments.
This feature requires generating DNS certificates and disabling Galley and the sidecar injector managing their own webhooks. Thus currently this feature can only be enabled and disabled on a new deployment.